lib/recurly/item.js

```diff
@@ -0,0 +1,23 @@
+import errors from './errors';
+
+const debug = require('debug')('recurly:item');
+
+/**
+ * Item
+ *
+ * Retrieves item pricing info based on an item code.
+ *
+ * @param {Object} options
+ * @param {String} options.itemCode
+ * @return {Promise}
+ */
+
+export default function item ({ itemCode } = {}) {
+  debug(itemCode);
+
+  if (typeof itemCode !== 'string') {
+    throw errors('invalid-option', { name: 'itemCode', expect: 'a String' });
+  }
+
+  return this.request.get({ route: `/items/${itemCode}`, cached: true });
```

Comment:
I went with a Promise API here despite most object getters being callback-oriented. In a few months I would like to migrate all other getters to return Promises and fulfill optional callbacks, in order to fully transition the object APIs. They are internal, but I suspect some folks do make direct use of them.